src/calendar/day/index.tsx

```diff
@@ -74,42 +52,46 @@ export default class Day extends Component<DayProps> {
       }
     }
     return label;
-  }
+  }, [marking]);
 
-  getAccessibilityLabel = memoize((day, marking, isToday) => {
+  const getAccessibilityLabel = useMemo(() => {
     const today = getDefaultLocale().today || 'today';
     const formatAccessibilityLabel = getDefaultLocale().formatAccessibilityLabel || 'dddd d MMMM yyyy';
-    const markingLabel = this.getMarkingLabel(marking);
-
-    return `${isToday ? today : ''} ${day.toString(formatAccessibilityLabel)} ${markingLabel}`;
-  });
 
-  getDayComponent() {
-    const {dayComponent, markingType} = this.props;
+    return `${_isToday ? today : ''} ${day?.toString(formatAccessibilityLabel)} ${markingLabel}`;
+  }, [day, marking, _isToday]);
 
+  const Component = useMemo(() => {
```

Comment:
No reason memoizing it, cause it just returns the component and not an instance of a component. 
The component is always the same because it is being imported at the top of the file.